docs/docs/development/local-toolchain/build-flash.mdx

```diff
@@ -95,40 +103,26 @@ west build -b nice_nano_v2 -- -DSHIELD=vendor_shield -DZMK_EXTRA_MODULES="C:/Use
 
 ### Building from `zmk-config` Folder
 
```

Comment:
I do agree adding documentation for [Permanent CMake Arguments](https://docs.zephyrproject.org/3.5.0/develop/west/build-flash-debug.html#id17) is most likely interesting for a lot of people. But I do think adding it at the marked location is not a good idea since most of the commands use such arguments. I think adding it as a general information at the top of the file is better. What do you think?

I do have to say I do not really understand what is mentioned in second link regarding the [west config](https://docs.zephyrproject.org/3.5.0/develop/west/config.html#west-config-cmd). Guess that is due to the fact that I am not that familiar with `west`. Would you mind explaining that to me?

Reply:
The west config note is just a minor thing. If we mention permanent cmake arguments then we need to mention west config, as the permanent arguments are set using west config.

Reply:
I did my best to write something appropriate. If anything is wrong or needs adjustment let me know.